internal/service/databaseuser/model_database_user.go

```diff
@@ -43,7 +42,13 @@ func NewMongoDBDatabaseUser(ctx context.Context, dbUserModel *TfDatabaseUserMode
 		Roles:        NewMongoDBAtlasRoles(rolesModel),
 		Labels:       NewMongoDBAtlasLabels(labelsModel),
 		Scopes:       NewMongoDBAtlasScopes(scopesModel),
-	}, nil
+	}
+
+	if statePasswordValue != dbUserModel.Password {
```

Comment:
i guess this also covers if the password attribute is deleted, then we set nil so it's not sent

Reply:
correct, added an additional unit test to cover this case as well, thanks